db/migrate/20190108161050_add_email_template_reference_to_responses.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailTemplateReferenceToResponses < ActiveRecord::Migration[5.2]
```

Comment:
Storing the template used in the database is essential in case we ever need to send the email again (error recovery situations)